wazo_acceptance/helpers/webrtc_phone.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2015-2025 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import docker
+from playwright.sync_api import Browser, Page
+from wazo_test_helpers import until
+
+from wazo_acceptance.helpers.phone import Phone
+
+CONTAINER_NAME = 'acceptance-webrtc-test-helper'
+EXPOSED_PORT = 3000
```

Comment:
Could port be randomized in the class? The way I read it now is that only 1 webrtc phone can be active at the same time